includes/core-changes/visualbasic/3.0/vbnewline-is-obsolete.md

```diff
@@ -1,22 +1,22 @@
 ﻿### Microsoft.VisualBasic.Constants.vbNewLine is obsolete
 
-The <xref:Microsoft.VisualBasic.Constants.vbNewLine?displayProperty=fullName> constant is marked [Obsolete](xref:System.ObsoleteAttribute) starting with .NET Core 3.0 Preview 8.
+The <xref:Microsoft.VisualBasic.Constants.vbNewLine?displayProperty=fullName> constant is marked as [obsolete](xref:System.ObsoleteAttribute) starting with .NET Core 3.0 Preview 8.
 
 #### Version introduced
 
 3.0 Preview 8
 
 #### Change description
 
-Starting with .NET Core 3.0 Preview 8, the [Obsolete](xref:System.ObsoleteAttribute) attribute has been applied to the <xref:Microsoft.VisualBasic.Constants.vbNewLine?displayProperty=fullName> constant. Use of the constant produces a compiler warning. In previous releases of both .NET Core and .NET Framework, it was not marked as obsolete.
+Starting with .NET Core 3.0 Preview 8, <xref:System.ObsoleteAttribute> has been applied to the <xref:Microsoft.VisualBasic.Constants.vbNewLine?displayProperty=fullName> constant. Use of the constant produces a compiler warning. In .NET Framework and previous releases of .NET Core, it was not marked as obsolete.
 
-This change was made to support Visual Basic as a language for multi-platform development. The `vbNewLine` constant is equivalent to `\r\n`, the newline character sequence on Windows. On Unix-based systems, the newline character is `\n`.
+This change was made to support Visual Basic as a language for multi-platform development. The <xref:Microsoft.VisualBasic.Constants.vbNewLine> constant is equivalent to `\r\n`, the newline character sequence on Windows. On Unix-based systems, the newline character is `\n`.
 
 #### Recommended action
 
-The [Obsolete](xref:System.ObsoleteAttribute) attribute message for `vbNewLine` includes the following recommendation:
+The <xref:System.ObsoleteAttribute> message for <xref:Microsoft.VisualBasic.Constants.vbNewLine> includes the following recommendation:
```

Comment:
Attributes are a bit tricky because usually they apply the short version of the attribute and I think it reads better in the original version.